test/Farmer/addFarm.test.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+import nock from "nock";
+import { describe, it } from "mocha";
+import { BASEURL } from "../../src/core/urls.js";
+import { farmSaveSuccess, farmSaveFailure } from "./farmerResponses.js";
+import { addFarm } from "../../src/users/farmer/farmmanagement.js";
+import { isTextOnly } from "../../src/users/farmer/farmermenus.js";
+
+describe("Add farm", () => {
+  beforeEach(() => {
+    nock(`${BASEURL}`).post("/ussd/farm/save").reply(200, farmSaveSuccess);
+    nock(`${BASEURL}`).post("/ussd/farm/save").reply(400, farmSaveFailure);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
    nock(`${BASEURL}`).post('/ussd/farm/save').reply(400, farmSaveFailure);
```